packages/senhub/src/hooks/useGotoApp.ts

```diff
@@ -26,12 +27,18 @@ export const useGoToAppCallback = () => {
   const visibleInstaller = useRootSelector(
     (state: RootState) => state.ui.visibleInstaller,
   )
+  const run = useRootSelector((state: RootState) => state.walkthrough.run)
+  const step = useRootSelector((state: RootState) => state.walkthrough.step)
 
   const onGotoAppCallback = useCallback(
     async ({ appId, blank = false, search }: GoToAppProps = {}) => {
       if (visible) await dispatch(setVisible(false))
       if (visibleActionCenter) await dispatch(setVisibleActionCenter(false))
       if (visibleInstaller) await dispatch(setVisibleInstaller(false))
+      if (run && step === 2)
```

Comment:
This logic never belongs to `useGoToApp`, please delete and move it to the source that calls `useGoToApp`